### ISSUE TITLE
Metatag for Twitter Title is appearing just for Custom Text

### ISSUE THREAD

**Issue by Robsondj** [nystudio107/craft-seomatic#1266]
### Describe the bug
The Twitter Title has appeared just when we configured SEO this way:
<img width="503" alt="image" src="https://user-images.githubusercontent.com/621503/212769907-c209de0b-a783-49d5-afd8-c4a32a1c2f23.png">
<br>
and the metatag is showing as expected
<img width="664" alt="image" src="https://user-images.githubusercontent.com/621503/212769678-e9f7ded1-a45d-4e2c-8dd4-8032bfdd54fc.png">
<br>
But in all the other options for this field, the Twitter Title metatag doesn't appear

### To reproduce
1. Go to Entries
2. Choose an Entry 
3. Go to the SEO tab
4. Go to the Twitter tab
5. Turn on the Override option for **Twitter Title Source**
6. Choose an option and save
7. Go to the website for this Entry and check if there is a metatag like this `<meta name="twitter:title" content="Twitter Title">`

### Expected behavior
All possible options in this field should put his value in the proper metatag

### Screenshots
<img width="791" alt="image" src="https://user-images.githubusercontent.com/621503/212771513-c9495058-d087-4f94-8a44-a78c12e28ff9.png">
<img width="801" alt="image" src="https://user-images.githubusercontent.com/621503/212771547-99b7422a-a1d5-4df9-b725-77fffaaa5261.png">
<img width="685" alt="image" src="https://user-images.githubusercontent.com/621503/212771703-f44aba6a-6bcb-4874-a44d-ea84efd804ac.png">
<img width="1061" alt="image" src="https://user-images.githubusercontent.com/621503/212771753-8a1c2b06-f5f5-49b9-b4f5-d6992979ebe8.png">

### Versions
- Plugin version: 4.0.18
- Craft version: 4.3.6

**Comment by khalwat** [comment]
Please update to SEOmatic 4.0.19 and let me know if you continue to see this issue.

**Comment by Robsondj** [comment]
Hey @khalwat, I updated to 4.0.19, and now even with Custom Text it doesn't work

**Comment by khalwat** [comment]
We are going to have to revisit this again -- there was a regression in the fix we did for you.

Something is very wrong with your setup, but I'm not sure what it is.

Please get me a database dump and your `composer.json` file so I can look into it locally.

andrew@nystudio107.com

**Comment by khalwat** [comment]
I was able to reproduce your issue locally with the composer.json that you sent along… but it has SEOmatic locked to this version:

"nystudio107/craft-seomatic": "4.0.18”,

This should really be set to the following, which will pull down the _latest_ SEOmatic:

"nystudio107/craft-seomatic": “^4.0.20”,

(Note the ^ in the server there)

In any event, after a bunch of debugging, I found out the problem:

You have defined a global set named ’SEOmatic’ in your Settings -> Globals -> SEOmatic. This overwrites the injected seomatic variable, causing it to fail to render. Renaming it to SEOmatic Settings (and the Handle to seomaticSettings) fixes the problem.
